Eatery/Views/Eateries/EateryScreenViews/MealStationTableViewCell.swift

```diff
@@ -33,19 +31,58 @@ class MealStationTableViewCell: UITableViewCell {
             make.leading.trailing.equalToSuperview().inset(10)
         }
 
+        collapseTitleLabelConstraint = titleLabel.snp.prepareConstraints({ make in
+            make.height.equalTo(0)
+        }).first
```

Comment:
```suggestion
        collapseTitleLabelConstraint = titleLabel.snp.prepareConstraints { make in
            make.height.equalTo(0)
        } .first
```
What exactly is `first` here? It's ambiguous to me on first glance. Can you either add a doc comment or access it in a clearer way?